episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Changing display settings
+
+What happens to pixel values when we change the display settings? Try changing 
+the contrast limits or colormap in the layer controls. You should see that the 
+histogram stays the same, no matter what settings you change i.e. the display 
+settings don't affect the underlying pixel values.
+
+This is one of the reasons it's important to use software designed for 
+scientific analysis to work with your light microscopy images. Software like 
+Napari and ImageJ will try to ensure that your pixel values remain unchanged, 
+while other image editing software (designed for working with photographs) may 
+change the pixel values in unexpected ways. Also, even with scientific software, 
+some image processing steps (that we'll see in later episodes) will change the 
+pixel values. 
+
+Keep this in mind and make sure you always retain a copy of your original data, 
+in its original file format! We'll see in episode X that original image files 
+contain important metadata that should be kept for future reference.
+
+## Colormaps / LUTs
+
+Let's dig deeper into Napari's colormaps. We've already seen in episode X that 
+the colormap can be changed by choosing different options from the 'colormap' 
+menu in the layer controls. But how do these colormaps actually work?
+
+Colormaps (also known as lookup tables or LUTs) match each pixel value to a 
+colour to use for display. For example, see the diagram below showing the 
+'gray' colormap:
+
+![](fig/gray-colorbar.png
+){alt="Grey colormap shown as a colorbar with corresponding pixel values" 
+width='50%'}
+
+Each pixel value from 0-255 maps to a corresponding colour. See the diagram 
+below for examples of 4 different colormaps with an example image histogram:
+
+![](fig/colorbar-comparison.png
+){alt="Four histograms showing the gray, green, viridis and inferno colormap 
+and correspoinding colorbar"}
+
+Why would we want to use different colormaps?
+
+- to highlight specific features in an image
+- to help with overlaying multiple images, as we saw in episode X when we 
+displayed green nuclei and magenta membranes together.
+- to help interpretation of an image. For example, if we used a red fluorescent 
+label in an experiment, then using a red colormap might help people understand 
+the image quickly.
+
+## Brightness and contrast
+
+As the final section of this episode, let's learn more about the 
+'contrast limits' in Napari. As we saw in episode X, adjusting the contrast 
+limits in the layer controls changes how bright different parts of the image are 
+displayed. What is really going on here though?
+
+In fact, the 'contrast limits' are adjusting how our colormap gets applied to 
+the image. For example, consider the standard gray colormap:
+
+![](fig/contrast-comparison-0-255.png
+){alt="Histogram coloured by gray colormap with corresponding colorbar. Contrast 
+limits 0 and 255." width='70%'}
```

Comment:
Which image do these histograms correspond to? Can we put up the associated image so that they can see the changes?

Reply:
These histograms correspond to the coins image - I'll add this into the plots